### ISSUE TITLE
Toggle auto auto-complete button

### ISSUE THREAD

**Issue by coristig** [yhat/rodeo#218]
Requested by several users:

Automatically trigger tab-completion w/o requiring tab to be hit

**Comment by boral** [comment]
Yes this feature is very important. I had also requested it here

https://github.com/yhat/rodeo/issues/214
